src/txkube/_swagger.py

```diff
@@ -780,6 +780,7 @@ def compare_pclass(self, other):
         content["serialize"] = _serialize_with_omit
         content["__new__"] = discard_constant_fields
         content["__cmp__"] = compare_pclass
+        content["__hash__"] = PClass.__hash__
```

Comment:
Is this a meaningful step?  I don't quite understand the problem.  The Python 3 docs say that a class with an `__eq__` will implicitly get a `None` __hash__` if it does not define one but this class doesn't have an `__eq__` does it?  It does have a `__cmp__` but that's not supported on Python 3...

Maybe this is an issue with the `__eq__` inherited from `PClass`?  But then why isn't `__hash__` inherited as well...

So.  I'm confused.  What does this change enable?  Shouldn't the `__cmp__` be fixed as well?